### PR TITLE
Spearbit Audit Changes

### PR DESCRIPTION
**Pull request by hexonaut**
Changes:

 * Remove the no-op role admin grants
 * Merge duplicate min grace period check
 * Spacing nit
 * Removed unused param in `_executeTransaction`
 * Better wording on `IExecutor` comment

### CODE REVIEW

File: src/Executor.sol [hunk 1]
```diff
@@ -40,16 +40,9 @@ contract Executor is IExecutor, AccessControl {
         uint256 delay_,
         uint256 gracePeriod_
     ) {
-        if (
-            gracePeriod_ < MINIMUM_GRACE_PERIOD
-        ) revert InvalidInitParams();
-
         _updateDelay(delay_);
```
Comment (lucas-manuel) [inline src/Executor.sol:43]: Original comment [here](https://github.com/spearbit-audits/review-makerdao/pull/3#discussion_r1721407340)

File: src/Executor.sol [hunk 2]
```diff
@@ -191,7 +182,7 @@ contract Executor is IExecutor, AccessControl {
     function getCurrentState(uint256 actionsSetId) public view override returns (ActionsSetState) {
         if (actionsSetCount <= actionsSetId) revert InvalidActionsSetId();
 
-        ActionsSet storage actionsSet =_actionsSets[actionsSetId];
+        ActionsSet storage actionsSet = _actionsSets[actionsSetId];
```
Comment (lucas-manuel) [inline src/Executor.sol:185]: Original comment [here](https://github.com/spearbit-audits/review-makerdao/pull/3#discussion_r1721421526)

File: src/Executor.sol [hunk 3]
```diff
@@ -208,7 +199,6 @@ contract Executor is IExecutor, AccessControl {
         uint256 value,
         string memory signature,
         bytes memory data,
-        uint256 executionTime,
```
Comment (lucas-manuel) [inline src/Executor.sol:211]: Original comment [here](https://github.com/spearbit-audits/review-makerdao/pull/3#discussion_r1721426107)

**Review by lucas-manuel — APPROVED** [review]
LGTM pending spearbit confirmation that audit is finished and that the don't want to do issue based PRs

**Review by barrutko — APPROVED** [review]
LGTM

**Comment by lucas-manuel** [comment]
This PR addresses the following Spearbit issues:

https://github.com/spearbit-audits/review-makerdao/issues/11
https://github.com/spearbit-audits/review-makerdao/issues/21
https://github.com/spearbit-audits/review-makerdao/issues/22

**Pull request merged**